apps/client/src/components/compounds/DashboardHeader/index.tsx

```diff
@@ -8,12 +13,53 @@ const DashboardHeader = ({
   onClick: () => void
   className?: string
 }) => {
+  const [isPopoverOpen, setIsPopoverOpen] = useState(false)
+
   return (
     <header className={classNames('container', styles.header, className)}>
       <button onClick={onClick}>T</button>
       <div>
-        <input type="text" />
-        <div>A</div>
+        <div className={styles.headerSearchInput}>
+          <SearchIcon className={styles.headerSearchIcon} />
+          <Input
+            className={styles.headerSearchPlaceholder}
+            type="text"
+            label={''}
+            placeholder={'Search'}
+          />
```

Comment:
make this a component, because this might get bigger in the future (like implement the search future for different use cases

Reply:
Ok